### PR TITLE
Updating Icm notifications

### PR DESCRIPTION
**Pull request by stevenernst131**
image

image

### CODE REVIEW

File: AppLensV2/app/Incident/incidentdirective.html [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
-                <md-button ng-click="incidentctrl.openServiceHealth()">Open Service Health Detector</md-button>
-                <md-button ng-href="{{incident.link}}" target="_blank">Open Icm Incident</md-button>
-            </md-card-actions>
-        </md-card>
-    </div>
+        <div ng-repeat="incident in incidentctrl.incidentNotifications" layout="column" flex="95" style="margin-top: 20px !important; margin-right: 15px; margin-left:15px;padding:0px;border-radius: 5px;background-color: white;color: #676767;border: 1px solid #ececec;background-color: white;border-left-width: 5px;border-left-color: rgb(244, 67, 54);box-shadow: 1px 1px 7px 0px #c5c3c3;">
+
+            <md-toolbar style="min-height: 50px;background-color:#F4F4F4;color:rgb(0,150,136);z-index:0;height: 30px;">
+                <div class="md-toolbar-tools">
+                    <h5 style="margin:auto 10px auto;font-weight:500;color: rgb(244, 67, 54);margin-top: 10px;margin-bottom: 10px;">{{incident.title}}</h5>
```
Comment (ShekharGupta1988) [inline AppLensV2/app/Incident/incidentdirective.html]: should we have a fixed Heading whenever there is a platform incident (LSI or CRI)?

Comment (stevenernst131) [reply AppLensV2/app/Incident/incidentdirective.html]: fixed heading? You mean static text? if you look at directive code, that is basically the case, just change it slightly if incident is active or not

Comment (hforeste) [inline AppLensV2/app/Incident/incidentdirective.html]: Whenever there is a "Past Live Site Incident" and I see that the incident title color is red and the containing div's left border color is red, I still get the impression that the incident is still happening. 

What do you think about changing the color according to the incident status, similar to how it is in IcM? It looks like you already have the incident status in the incident notification object.

Active: Red
Mitigated: Yellow
Resolved: Green

I feel like the CSS folks eyes are trained to see red as active and the message of "Past live incident" will confuse them.

File: AppLensV2/app/Incident/incidentdirective.html [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
 
     <div layout="row" ng-if="!incidentctrl.loadingCRIs && incidentctrl.criIncidentNotifications && incidentctrl.criIncidentNotifications.length > 0">
         <div layout="column" flex="5" layout-align="center center"></div>
-        <md-card flex="95" md-theme="incident-theme" ng-repeat="incident in incidentctrl.criIncidentNotifications">
-            <div class="incident-message-container" ng-bind-html="incident.message">
+        <div ng-repeat="incident in incidentctrl.criIncidentNotifications" layout="column" flex="95" style="margin-top: 35px !important; margin-right: 15px; margin-left:15px;padding:0px;border-radius: 5px;background-color: white;color: #676767;border: 1px solid #ececec;background-color: white;border-left-width: 5px;border-left-color: rgb(244, 67, 54);box-shadow: 1px 1px 7px 0px #c5c3c3;">
+
+            <md-toolbar style="min-height: 50px;background-color:#F4F4F4;color:rgb(0,150,136);z-index:0;height: 30px;">
+                <div class="md-toolbar-tools">
+                    <h5 style="margin:auto 10px auto;font-weight:500;color: rgb(244, 67, 54);margin-top: 10px;margin-bottom: 10px;">{{incident.title}}</h5>
```
Comment (hforeste) [inline AppLensV2/app/Incident/incidentdirective.html]: Same thing here as mentioned above

File: AppLensV2/app/Incident/incidentdirective.html (hunk 1 above)
Comment (stevenernst131) [reply AppLensV2/app/Incident/incidentdirective.html]: good feedback

**Comment by ShekharGupta1988** [comment]
Merging this for now. Steve will be doing the changes on top of it

**Pull request merged**